bundles/org.openhab.binding.somfytahoma/src/main/resources/OH-INF/config/config.xml

```diff
@@ -69,5 +69,35 @@
 			<description>Specifies the delay in milliseconds between subsequent retries after a command failure</description>
 			<default>1000</default>
 		</parameter>
+
+		<parameter name="devMode" type="boolean" required="false">
```

Comment:
You need to update the default translations file (.properties) to add these new parameters.

Reply:
of course, added

Reply:
You forgot the token parameter.

Reply:
sorry, added